AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -220,44 +215,36 @@ public async Task<IActionResult> CloneHearing(Guid hearingId, MultiHearingReques
             }
             catch (BookingsApiException e)
             {
-                _logger.LogError(e,
-                    "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
+                _logger.LogError(e, "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
                     hearingId, e.StatusCode, e.Response);
-                if (e.StatusCode != (int) HttpStatusCode.NotFound)
-                    throw;
-
+                if (e.StatusCode != (int) HttpStatusCode.NotFound) throw;
                 return NotFound($"No hearing with id found [{hearingId}]");
             }
 
             try
             {
                 var newParticipantList = new List<ParticipantRequest>();
-
-                if (CheckHearingWithinThirtyMinutes(originalHearing))
+                if (IsHearingStartingStartSoon(originalHearing) && originalHearing.Status == BookingStatus.Created &&
+                    !_hearingsService.IsAddingParticipantOnly(request, originalHearing))
+                {
+                    _logger.LogWarning("You can't edit a hearing within 30 minutes of it starting");
+                    ModelState.AddModelError(nameof(hearingId),
+                        $"You can't edit a hearing [{hearingId}] within 30 minutes of it starting");
```

Comment:
Use a variable for the string if these are going to be the same.

Also set 30 as a readonly property to be used here and also line 308.